### PR TITLE
Sql cleanup

### PR DESCRIPTION
**Pull request by pnorman**
more of the same.

### CODE REVIEW

File: project.yaml [hunk 1]
```diff
@@ -1315,11 +1343,16 @@ Layer:
...
-              ) AS placenames_capital
+      table: |-
+        (SELECT
+            way,
+            place,
+            name,
+            ref
+          FROM planet_osm_point
+          WHERE place IN ('city', 'town')
+            AND capital IN ('yes', '4')
```
Comment (pnorman) [inline project.yaml]: This seems odd. Should this include 2 and 3?

**Comment by math1985** [comment]
The Travis CI build failed.

File: project.yaml (hunk 1 above)
Comment (math1985) [inline project.yaml]: I think we shouldn't include 2. Normally capital=2 is tagged as capital=yes. There are only 2 objects with capital=2 in the database.

We could include capital=3, but it's not urgent as there are only 8 instances in the database.

**Comment by pnorman** [comment]
fixed

**Pull request merged**